flake.nix

```diff
@@ -3,7 +3,7 @@
 
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
-    nixpkgs-unstable.url = "github:nixos/nixpkgs/632f04521e847173c54fa72973ec6c39a371211c";
```

Comment:
`nixpkgs-unstable` in flake.lock already refers to the specific hash, so there is no need to specify the same hash.